### DISCUSSION TITLE
Is `M-x` not supposed to be `vertico-repeat`-able?

### DISCUSSION THREAD

**Discussion by leungbk** [minad/vertico#424] [Q&A] [answered]
With a config like

```elisp
(use-package vertico
  :straight ( :host github
              :repo "minad/vertico"
              :files (:defaults "extensions/*")
              :includes (vertico-repeat))
  :demand t
  :config
  (define-key global-map (kbd "C-c C-r") #'vertico-repeat)
  (vertico-mode 1))
```

I find that after I `M-x` and exit the session selecting (say) `next-line` from the list of completions, I am unable to recover that session with `vertico-repeat` and am told "No repeatable session".

**Comment by minad** [comment, marked as answer]
It is excluded by default, but this can be changed via a customization variable.
